src/components/dev/LocalizationReport.js

```diff
@@ -65,22 +69,27 @@ class LocalizationReport extends Component {
     modalOpen:        false,
     modelLocKey:      null,
     compareLocKey:    null,
-    filter:           'false',
+    filter:           `false`,
```

Comment:
Also should discuss quoting conventions; most projects I see use single quotes for anything without interpolation, backticks only if it's interpolating or literals with newlines. I (marginally) prefer defaulting to double quotes, but I admit it doesn't make much difference.